.eleventy.js

```diff
@@ -34,25 +34,37 @@ module.exports = function (config) {
     config.addTransform("parse", parseTransform);
 
     // Passthrough copy
+    config.addPassthroughCopy({"src/admin/config.yml": "admin/config.yml"});
     config.addPassthroughCopy({"src/assets/icons": "/"});
     config.addPassthroughCopy({"src/assets/images": "assets/images"});
     config.addPassthroughCopy({"src/posts/images": "posts/images"});
 
+    config.addNunjucksFilter("getSiblingValue", function (value, inputKey, inputValue, siblingKey) {
+        let siblingValue = false;
```

Comment:
Clearer here is 

let matchedItem = value.find(item => item[inputKey] === inputValue);
return matchedItem && matchedItem[siblingKey];

You might appreciate some JSDocs here in case you forget someday what this function does - terribly easy to do!

Reply:
Indeed, JSDoc throughout the project would be excellent! We could file an issue for that and tackle it separately, no need to add everything with this PR.